apps/server/src/shared/repo/user/user.repo.integration.spec.ts

```diff
@@ -114,6 +128,7 @@ describe('user repo', () => {
 			userA = userFactory.build({ school, externalId: '111' });
 			userB = userFactory.build({ externalId: '111' });
 			await em.persistAndFlush([userA, userB]);
+			em.clear();
```

Comment:
Right, but we need `em.clear()` inside the test after the test setup and before fetching the data again from the db. Otherwise we would just compare our expectations against the setup data inside the Unit of Work. That is really important!